dao/src/main/java/se/fortnox/reactivewizard/db/Schema.java

```diff
@@ -0,0 +1,12 @@
+package se.fortnox.reactivewizard.db;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.PARAMETER)
+public @interface Schema {
```

Comment:
We should drop this as well I think

Reply:
I actually did remove that, but borked my workspace and lost the change. Thanks for catching it!